File: Code/Websites/DanpheEMR/Services/NewClinical/DTOs/Questionary_DTO.cs
using DanpheEMR.Controllers.Clinical_New.DTO;

namespace DanpheEMR.Services.NewClinical.DTOs
{
	public class Questionary_DTO
	{
		public int ClinicalInformationId { get; set; }
		public int PatientId { get; set; }
		public int PatientVisitId { get; set; }
		public int ClinicalHeadingId { get; set; }
		public int? ParentHeadingId { get; set; }
		public int FieldId { get; set; }
		public string InputType { get; set; }
		public string FieldValue { get; set; }
		public string Remarks { get; set; }
		public QuestionaryData_DTO QuestionaryData { get; set; }
	}
}

File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/clinical-new/shared/dto/questionary.dto.ts
import { QuestionarySingleSelectMultipleSelect } from "./questionary-singleselect-multiple-select.dto";
import { QuestionaryTextBoxFreeTypeNumber } from "./questionary-textbox-freetype-number.dto";

export class Questionary {
    ClinicalInformationId: number;
    PatientId: number;
    PatientVisitId: number;
    ClinicalHeadingId: number;
    ParentHeadingId?: number;
    FieldId: number;
    InputType: string;
    FieldValue: string;
    Remarks: string;
    QuestionaryData: {
        textBoxFreeTypeNumber: QuestionaryTextBoxFreeTypeNumber[];
        singleSelectMultipleSelect: QuestionarySingleSelectMultipleSelect[];
    };
}
